src/praisonai/praisonai/bots/botos.py

```
@@ -1061,6 +1061,35 @@ def _resolve_env(val):
+        """
+        from praisonaiagents.bots import ProbeResult
+
+        if not self._bots:
+            return {}
+
+        platforms = list(self._bots.keys())
+
+        async def _probe_one(bot: Bot) -> Any:
+            try:
+                return await bot.probe()
+            except Exception as e:  # pragma: no cover — defensive
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)